contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+# Proposal to Standardize Development Process
+
+## Links
+```
+---
+Links:
+- Issues:
+  - [someIssueURL]()
+- PRs:
+  - https://github.com/kubernetes/community/pull/967
+- MailingListDiscussions:
+  - https://groups.google.com/forum/#!topic/kubernetes-dev/65A-3ULYPB0
+- Documentation:
+  - [someDocsLinkURL]()
+```
+
+## Responsible SIG(s)
+
+- SIG Release
+- SIG PM
+- SIG Architecture
+- SIG Governance
+- Technical Steering Committee
+
+## Summary
+
+A standardized development process for Kubernetes is proposed in order to:
+- provide a common structure for proposing changes to Kubernetes
+- ensure that the motivation for a change is clear
+- allow for the enumeration stability milestones and stability graduation
+  criteria
+- persist project information in a Version Control System (VCS) for future
+  Kubernauts
+- support the creation of _high value user facing_ information such as:
```

Comment:
There are competing goals here that I'm not sure belong in the same unified process.  Proposing a change and locking down the implications of that is often a multi-version slog.  I would expect a proposal for something major like, say, moving to an aggregated API model.  That doesn't lend itself to release notes and the release announcement.

While I agree that we want a way to track what is going in to a release (and have those notes point to these proposals!) I don't think that these proposals will line up neatly with releases.